docs/REST API/rest-api.md

```diff
@@ -4535,6 +4535,13 @@ For example, a policy might ask “Is Gatekeeper enabled on macOS devices?“ Th
 
 `GET /api/v1/fleet/global/policies`
 
+#### Parameters
+
+| Name                    | Type    | In    | Description                                                                                                                                                                                                                                                                                                                                 |
+| ----------------------- | ------- | ----- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| page                    | integer | query | Page number of the results to fetch.                                                                                                                                                                                                                                                                                                        |
+| per_page                | integer | query | Results per page.       
+
```

Comment:
This part lgtm 👍 

For the "List team policies" endpoint, the table [here](http://localhost:2024/docs/rest-api/rest-api#list-team-policies) will need to be updated as well. (If the change is only for global policies, feel free to dismiss this review and I'll approve ✅)

Reply:
great catch!  updated